.travis.yml

```diff
@@ -1,19 +1,10 @@
 sudo: required
-dist: trusty
-language: node_js
-cache:
```

Comment:
Is this not needed anymore? :-O

Reply:
As of very recently, it's done by default: https://docs.travis-ci.com/user/caching/#npm-cache